packages/bs-reform/src/ReForm.re

```diff
@@ -414,18 +468,52 @@ module Make = (Config: Config) => {
         });
     };
 
+    let getFieldErrorState = field =>
+      getFieldState(field)
+      |> (
+        fun
+        | Error(error) => Error(error)
+        | NestedErrors(errors) => Errors(errors)
+        | Pristine
+        | Valid => Ok
+      );
+
+    let getNestedFieldError = (field, index, name) =>
+      getFieldState(field)
+      |> (
+        fun
+        | NestedErrors(errors) => {
+            switch (
+              errors
+              |> Js.Array.find(error =>
```

Comment:
if `errors` are sorted by index we could access via `Belt.Array.get`

Reply:
Agreed.